Sources/Sentry/SentrySpan.m

```diff
@@ -77,6 +78,13 @@ - (void)finishWithStatus:(SentrySpanStatus)status
     [self finish];
 }
 
+- (SentryTraceHeader *)toTraceHeader
+{
+    return [[SentryTraceHeader alloc] initWithTraceId:self.context.traceId
+                                               spanId:self.context.spanId
+                                       sampleDecision:self.context.sampled];
```

Comment:
Just to confirm, `context` is immutable and we don't expose a way to chagne the instance we hold here?

Reply:
Yes, context is immutable.